gtsfm/utils/triangulation.py

```diff
@@ -0,0 +1,42 @@
+"""Utilities to support triangulation.
+
+Authors: Ayush Baid
+"""
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler, Unit3
+
+import gtsfm.utils.geometry_comparisons as geometry_utils
+
+
+def calculate_triangulation_angle_in_degrees(
```

Comment:
I think this could be in geometry_utils too, unless you think we would have more functions here. A util file just for one function would be odd.